expression/expression.go

```diff
@@ -1055,7 +1055,7 @@ func scalarExprSupportedByFlash(function *ScalarFunction) bool {
 
 		ast.Sqrt, ast.Log, ast.Log2, ast.Log10, ast.Ln, ast.Exp, ast.Pow, ast.Sign,
 		ast.Radians, ast.Degrees, ast.Conv, ast.CRC32,
-		ast.JSONLength,
+		ast.JSONLength, ast.Repeat,
```

Comment:
Please also add an integration test. Like this: https://github.com/pingcap/tidb/pull/35872/files#diff-fa42dbc5f88d5aef4ee1cb7452ee4497fc545324b76bee0e44f48e5f1af9f512R2729.

Reply:
Done